torchvision/prototype/transforms/_geometry.py

```diff
@@ -225,6 +225,19 @@ def _check_fill_arg(fill: Union[int, float, Sequence[int], Sequence[float]]) ->
         raise TypeError("Got inappropriate fill arg")
 
 
+def _check_padding_arg(padding: Union[int, Sequence[int]]) -> None:
+    if not isinstance(padding, (numbers.Number, tuple, list)):
+        raise TypeError("Got inappropriate padding arg")
+
+    if isinstance(padding, (tuple, list)) and len(padding) not in [1, 2, 4]:
+        raise ValueError(f"Padding must be an int or a 1, 2, or 4 element tuple, not a {len(padding)} element tuple")
+
+
+def _check_padding_mode_arg(padding_mode: Literal["constant", "edge", "reflect", "symmetric"]) -> None:
```

Comment:
Nit: We should consider the use of enums here; this a textbook case for them. Using `torchvision._utils.StrEnum` will enable us to have the best of both worlds (strings and enums): staying aligned with Core while at the same time giving the option to our users to use idioms friendly to static analysis.

Reply:
Let me do that in a follow-up PR

Reply:
Could you just add a TODO comment?